core/src/main/scala/com/github/andyglow/json/JsonFormatter.scala

```diff
@@ -63,4 +63,4 @@ class JsonFormatter(step: Int = 2) {
   }
 }
 
-object JsonFormatter extends JsonFormatter(step = 2)
\ No newline at end of file
+object JsonFormatter extends JsonFormatter(step = 2)
```

Comment:
should it be included?

Reply:
it's only a matter of intellij adding a newline automatically..

Reply:
reverted